### ISSUE TITLE
0.7.8 and 0.7.7 exception: pymysql.err.InternalError: Packet sequence number wrong - got 142 expected 1

### ISSUE THREAD

**Issue by Zeusw** [PyMySQL/PyMySQL#512]
code：
```
import pymysql.cursors

connection = pymysql.connect(host='localhost',
                             user='root',
                             password='123456',
                             db='wikiurl',
                             charset='utf8mb4')

try:
    with connection.cursor() as cursor:
        sql = "select urlname, urlhref from urls where id is not null"
        conut = cursor.execute(sql)
        print(conut)
finally:
    connection.close()
```

This code in 0.7.6 is OK, but in 0.7.7 or 0.7.8 is not ok.
I think is, `id is not null ` in 0.7.7 is not supported.

**Comment by methane** [comment]
It seems duplicate of #511, but this is more easy to reproduce.
Thanks!

**Comment by methane** [comment]
Hmm, I can't reproduce.
"id is not null" can be used in PyMySQL 0.7.8.

mysqldump is required to reproduce.

**Comment by Zeusw** [comment]
thank you for your reply.
I found the problem.
`
INSERT INTO `wikiurl`.`urls2` (`id`, `urlname`, `urlhref`) VALUES ('137', '', 'https://en.wikipedia.org/wiki/Main_Page');
`
if urlname is '',There will be an exception:pymysql.err.InternalError: Packet sequence number wrong - got 6 expected 1

Note: Is Null character, not NULL.

**Comment by methane** [comment]
Could you show `show create table wikiurl`?

**Comment by Zeusw** [comment]
id  type:int  length:11 Primary Key
urlname type:varchar length:255
urlhref type:varchar length:1000

**Comment by Zeusw** [comment]
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Table | Create Table                                                                                                                                                                                                            |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| urls  | CREATE TABLE `urls` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `urlname` varchar(255) DEFAULT NULL,
  `urlhref` varchar(1000) DEFAULT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=306 DEFAULT CHARSET=utf8mb4 |
+-------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

**Comment by methane** [comment]
fixed via #513
